src/Patronage.DataAccess/Services/CommentService.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Patronage.Contracts.Helpers;
+using Patronage.Contracts.Interfaces;
+using Patronage.Contracts.ModelDtos.Comments;
+using Patronage.Models;
+
+namespace Patronage.DataAccess.Services
+{
+    public class CommentService : ICommentService
+    {
+        private readonly TableContext _dbContext;
+
+        public CommentService(TableContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public async Task<PageResult<CommentDto>?> GetAllCommentFromIssue(FilterCommentDto filter)
+        {
+            var baseQuery = _dbContext
+                .Comment
+                .Where(x => x.IssueId == filter.IssueId)
+                .AsQueryable();
+
+            if (!baseQuery.Any())
+            {
+                return null;
+            }
+
+            var totalItemCount = baseQuery.Count();
+
+            var comments = baseQuery
+                .Skip(filter.PageSize * (filter.PageNumber - 1))
+                .Take(filter.PageSize);
+
+            var items = await comments.ToArrayAsync();
```

Comment:
```suggestion
            var items = await comments.Select(x => new CommentDto(x).ToListAsync();
```
Instead of foreach loop. (Check if it works first, I'm just suggesting different approach)

Reply:
Done.